core/src/test/scala/scala/pickling/pickler/FastTypeTagTest.scala

```diff
@@ -0,0 +1,17 @@
+package scala.pickling.pickler
+
+import org.scalatest.FunSuite
+import scala.pickling._, Defaults._, json._
```

Comment:
It's a good point, but actually we need to test the inverse.  WE need to check that if people use the WillRobinson algorithm (which is the one that handled FastTypeTag) they will still get our staticly-donated picklers.   i.e. if I add `import static._` I might mask a failure case.